bpdm-pool-api/src/main/kotlin/org/eclipse/tractusx/bpdm/pool/api/PoolBpnApi.kt

```diff
@@ -62,4 +63,17 @@ interface PoolBpnApi {
     @Tag(name = ApiCommons.BPN_NAME, description = ApiCommons.BPN_DESCRIPTION)
     @PostMapping(CommonApiPathNames.SUBPATH_SEARCH)
     fun findBpnsByIdentifiers(@RequestBody request: IdentifiersSearchRequest): ResponseEntity<Set<BpnIdentifierMappingDto>>
+
+    @Operation(
+        summary = "Return BPNL/S/A based on the requested identifiers",
+        description = "Find business partner numbers by requested-identifiers."
+    )
+    @ApiResponses(
+        value = [
+            ApiResponse(responseCode = "200", description = "Found bpn to based on the requested identifiers"),
+        ]
+    )
+    @Tag(name = ApiCommons.BPN_NAME, description = ApiCommons.BPN_DESCRIPTION)
+    @PostMapping("/request-ids")
```

Comment:
This is a POST search endpoint. In order to align with the other Pool endpoints please use the '/search' suffix:
'/request-ids/search'

Reply:
Endpoint has been updated.